packages/graphql/src/entities/index.ts

```diff
@@ -1,27 +1,20 @@
-export * from './App'
+/* eslint-disable padding-line-between-statements */
+// created by autobarrel, do not modify directly
 
+export * from './App'
+export * from './Browser'
 export * from './DashboardProject'
-
-export * from './Project'
-
+export * from './LocalProject'
 export * from './Mutation'
-
+export * from './NavigationItem'
 export * from './NavigationMenu'
-
-export * from './LocalProject'
-
-export * from './ResolvedConfig'
-
+export * from './Project'
 export * from './Query'
-
+export * from './ResolvedConfig'
 export * from './TestingTypeInfo'
-
 export * from './Viewer'
-
 export * from './Wizard'
-
 export * from './WizardBundler'
-
 export * from './WizardFrontendFramework'
-
 export * from './WizardNpmPackage'
+export * from './run/'
```

Comment:
Is this a bug in autobarrel?
```suggestion
export * from './run'
```

Reply:
No, this is intentional. Trailing slash visually indicates that it's a directory